classes/order/Order.php

```diff
@@ -746,8 +746,15 @@ protected function setProductCustomizedDatas(&$product, $customized_datas)
      */
     protected function setProductCurrentStock(&$product)
     {
-        $product['current_stock'] = StockAvailable::getQuantityAvailableByProduct($product['product_id'], $product['product_attribute_id'], (int) $this->id_shop);
-        $product['location'] = StockAvailable::getLocation($product['product_id'], $product['product_attribute_id'], (int) $this->id_shop);
+        if (Configuration::get('PS_ADVANCED_STOCK_MANAGEMENT')
```

Comment:
```suggestion
        if (Configuration::get('PS_ADVANCED_STOCK_MANAGEMENT')
```

PS_ADVANCED_STOCK_MANAGEMENT Has been completely removed, you can keep only the else part

Reply:
Indeed

Reply:
@tleon This was the change in the method. ;-)

https://github.com/PrestaShop/PrestaShop/commit/74d5c5c924350feab0e6b3707bdb1e9488252ccf

Just cast location to string.